WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -206,7 +208,8 @@ + (NSArray *)routes
 
 + (id<FBResponsePayload>)handleDoubleTapCoordinate:(FBRouteRequest *)request
 {
-  XCUICoordinate *doubleTapCoordinate = [self.class getGestureCoordinate:request];
+  CGPoint doubleTapPoint = CGPointMake((CGFloat)[request.arguments[@"x"] doubleValue], (CGFloat)[request.arguments[@"y"] doubleValue]);
+  XCUICoordinate *doubleTapCoordinate = [self.class gestureCoordinateWithCoordinate:doubleTapPoint application:request.session.application shouldApplyOrientationWorkaround:YES];
```

Comment:
Just to summarise what we know:
1) On iOS 9.x coordinates are not inverted
2) On iOS 10.x coordinates are inverted
3) We can assume that coordinates passed by user are correct (meaning they are in reference to current orientation). This is mainly because we give them the coordinates or they manually calculate them.
Is that right?